src/core/utils.js

```diff
@@ -722,6 +722,19 @@ export function sanitizeUrl(url) {
   return braintreeSanitizeUrl(url)
 }
 
+export function isAbsoluteUrl(url) {
```

Comment:
**Change request**: new util functions need accompanying tests - ideally that would include cases for expected outputs, graceful handling of malformed inputs, and basic type safety (esp. `undefined`/`null`).